AUTHORING_GUIDE.md

```diff
@@ -467,17 +511,22 @@ To run a specific test from a specific following:
 nox -s py-3.7 -- snippets_test.py:test_list_blobs
 ```
 
-### Test Environment Setup
+### Running tests with Docker
 
-Because all tests are system tests that use live resources, running tests
-requires a Google Cloud project with billing enabled, as covered under
-[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+If you have Docker installed, you can use `scripts/run_tests_local.sh`
```

Comment:
nit: Can we condense this and "Docker setup" for brevity? 
```suggestion
If you have [Docker](https://www.docker.com/) installed and runnable by the local user, you can use `scripts/run_tests_local.sh`
```

Reply:
Done